### PR TITLE
add tolerations, affinity and nodeSelector to stackdriver-exporter

### LINKED ISSUE #11853: add support for tolerations, affinity and nodeSelector to stackdriver-exporter
**Is this a request for help?**:
No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

I would like the stackdriver-exporter chart to support tolerations, affinity and nodeSelector.

**Version of Helm and Kubernetes**:
helm "v2.12.3"
kubectl 1.10.11

**Which chart**:
stackdriver-exporter version 0.0.6

**What happened**:
No support for tolerations, affinity and nodeSelector.

**What you expected to happen**:
N/A

**How to reproduce it** (as minimally and precisely as possible):
Apply the chart.

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by baboune**
#### What this PR does / why we need it:

Fixes #11853.

Add support for tolerations, affinity and nodeSelect to stackdriver-exporter chart v0.0.6

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #11853

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [ x] Variables are documented in the README.md

### CODE REVIEW

**Comment by baboune** [comment]
Hi,
Could I get a review?

Thanks

File: stable/stackdriver-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Stackdriver exporter for Prometheus
 name: stackdriver-exporter
-version: 0.0.6
+version: 0.0.7
```
Comment (cpanato) [inline stable/stackdriver-exporter/Chart.yaml]: lets bump this to 1.0.0 since this chart is in stable folder

```suggestion
version: 1.0.0
```

**Review by cpanato — APPROVED** [review]

**Comment by baboune** [comment]
thank you

**Pull request merged**